pkg/cmd/properties/properties.go

```diff
@@ -288,13 +288,30 @@ func (p *properties) StringFlagValue(name string) (bool, string) {
 }
 
 func (p *properties) DurationPropertyValue(name string) time.Duration {
+	lowerLimit, _ := time.ParseDuration("30s")
+
 	s := p.parseStringValue(name)
 	d, _ := time.ParseDuration(s)
 
+	// Get config value and check if duration is less than 30s
+	if d < time.Second*30 {
+		flagName := p.nameToFlagName(name)
+		flag := p.rootCmd.Flag(flagName)
+
+		// since config value is < 30s, get duration default value
+		defaultValue := flag.DefValue
+		d, _ = time.ParseDuration(defaultValue)
+
+		if d > time.Second*30 {
```

Comment:
Can you set lowerLimit := time.Second * 30 and additionally use that var in the 2 conditionals?

Reply:
Yes 🤦‍♂️